cfme/fixtures/ansible_fixtures.py

```diff
@@ -18,12 +18,23 @@ def wait_for_ansible(appliance):
 
 
 @pytest.fixture(scope="module")
-def ansible_repository(appliance, wait_for_ansible):
+def ansible_repository(request, appliance, wait_for_ansible):
+    """
+    By default cfme_data.ansible_links.playbook_repositories.embedded_ansible is set for the url,
+    but you can specify it explicitly with @pytest.mark.parametrize decorator on your test function.
+
+    Example:
+    @pytest.mark.parametrize('ansible_repository', ['nuage'], indirect=True)
+    def test_function(ansible_repository):
+        ...
+    """
     repositories = appliance.collections.ansible_repositories
+    playbooks = cfme_data.ansible_links.playbook_repositories
```

Comment:
I'm going to push a commit tweaking how this is referenced.

Its important that this attribute lookup chain is within the try block below, because a cfme_data file without `ansible_links` or `playbook_repositories` maps will cause an exception during the fixture execution.